scijava/scijava-ops-engine/src/test/java/org/scijava/ops/engine/impl/ProvenanceTest.java

```diff
@@ -260,15 +288,16 @@ private OpInfo singularInfoOfName(String name) {
 
 	@Test
 	public void testAdaptatorWithDependencies() {
-		Function<Double[], Double[]> f = ops.op("test.provenanceComputer").inType(Double[].class).outType(Double[].class).function();
+		Function<Double[], Double[]> f = ops.op("test.provenanceComputer").inType(
+			Double[].class).outType(Double[].class).function();
 		@SuppressWarnings("unchecked")
 		InfoChain chain = ((RichOp<Function<Double[], Double[]>>) f).infoChain();
 		String signature = chain.signature();
 		Nil<Function<Double[], Double[]>> special = new Nil<>() {};
-		Nil<Double[]> inType = Nil.of(Double[].class);
-		Nil<Double[]> outType = Nil.of(Double[].class);
-		Function<Double[], Double[]> actual = ops.env().opFromID(signature, special, new Nil[] {inType}, outType);
-		Double[] apply = actual.apply(new Double[] {1., 2., 3.});
+		Function<Double[], Double[]> actual = ops.env().opFromSignature(signature,
+			special);
+		@SuppressWarnings("unused")
+		Double[] apply = actual.apply(new Double[] { 1., 2., 3. });

```

Comment:
Dontcha wanna assert the result? Or really... instead of calling `apply`, assert that the `actual` object is (or wraps) the expected function?

Reply:
scijava/scijava#83

Reply:
Resolving this in favor of that issue